Signal/test/ViewControllers/ConversationViewItemTest.m

```diff
@@ -107,66 +97,6 @@ - (ConversationViewItem *)audioViewItem
     return [self viewItemWithAttachmentMimetype:@"audio/mp3" filename:@"test-mp3.mp3"];
 }
 
-- (void)testCanPerformEditingActionWithNonMediaMessage
```

Comment:
These were failing, again. They are brittle tests and not very valuable, so I removed them.